templates/observability/what-is-observability.html

```diff
@@ -0,0 +1,165 @@
+{% extends "observability/base_observability.html" %}
+
+{% block title %}What is observability{% endblock %}
+
+{% block meta_description %}Observability is fundamental for applications and systems. Learn how it relates with monitoring and the different types of telemetry that is collected and analyzed in the state of the art of open source observability.{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1eoqDQzFiRJY2u_EO4JBuLBLqae4K3nrEf3qd88KHw0Y/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip u-no-padding--top u-no-padding--bottom">
+    <div class="p-strip--suru-topped">
+        <div class="row u-equal-height">
+            <div class="col-8">
+                <h1>What is observability?</h1>
+                <p>Observability is a crucial property of well-designed software systems. Your system is observable if you can know whether it is working as desired and, if it is not, who is impacted and how badly, what the issues are and where they are originating from.</p>
+            </div>
+            <div class="col-4 u-align--center u-vertically-center">
+                {{ image (
+                    url="https://assets.ubuntu.com/v1/db240da7-K8s_management_platform-01.svg",
+                    alt="",
+                    width="350",
+                    height="250",
+                    hi_def=True,
+                    loading="lazy"
+                    ) | safe
+                }}
+            </div>
+        </div>
+    </div>
+</section>
+
+<section class="p-strip--light">
+    <div class="row">
+        <div class="col-8">
+            <h2>Observability versus monitoring</h2>
+            <h3 class="p-heading--4">Rather, observability <em>through</em> monitoring</h3>
```

Comment:
not your fault but could you remove the italics please? Conversation [here](https://chat.canonical.com/canonical/pl/bqqyrj7b3ibwifat4eefit3zoh)